### PR TITLE
chore(textures): update what's new in basis_universal

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
Fixes are taken from here - https://github.com/BinomialLLC/basis_universal/commit/7a2094b807fcd8eb8e505d8c48a19b0690fd3404

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ Target Release Date: Q1 2022.
 
 - The `WKBWriter` supports options within a `wkb` key, instead of at the top level.
 
+**@loaders.gl/textures**
+- Fixed issues:
+  - Overflow issue on extreme colors in color_distance().
```
Comment (ibgreen) [inline docs/whats-new.md:57]: Are these the detailed release notes from basis? I think we can drop these here and just link to them. They seem to be very specialized.

File: docs/whats-new.md [hunk 2]
```diff
@@ -52,6 +52,13 @@ Target Release Date: Q1 2022.
...
+  - Explicitly disabling unaligned addressing in miniz and transcoder when UBSAN is enabled
+  - Disabling a needless optimization in transcoder causing ubsan issues 
+- Upgrade `basis_universal` libraries to [v1.16.3](https://github.com/BinomialLLC/basis_universal/releases/tag/1.16.3).
```
Comment (ibgreen) [inline docs/whats-new.md:60]: I think this line is all we need. Of course if we also support experimental worker writers etc we may want to mention that (separate PR). @belom88

**Review by ibgreen — APPROVED** [review]

File: docs/whats-new.md (hunk 1 above)
Comment (dryabinin-actionengine) [reply docs/whats-new.md]: Rmoved

File: docs/whats-new.md (hunk 2 above)
Comment (dryabinin-actionengine) [reply docs/whats-new.md:56]: Keep only this line.

**Pull request merged**